iphone/Classes/DeMarcelpociotCollectionviewCollectionView.h

```diff
@@ -8,7 +8,6 @@
 #import "TiUIView.h"
 #import "DeMarcelpociotCollectionviewCollectionViewProxy.h"
 #import "CHTCollectionViewWaterfallLayout.h"
-#import "M13ContextMenu.h"
```

Comment:
Your point seems relevant, let leave the lib out as is in the PR and continue. Thank for the help!